### PR TITLE
updated the file upload with the requested changes

### PR DESCRIPTION
**Pull request by jeremiahbooth-insight**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

fixed the file inputs to accept images and pdf's created the ability so see the currently uploaded files.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] API unit tests
- [ ] UI unit tests

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings or errors
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: UI/libs/core-public/src/lib/components/form-stepper/form-steps/FileUploadStep.vue [hunk 1]
```diff
@@ -5,6 +5,21 @@
       ref="form"
       v-model="state.valid"
     >
+      <v-subheader class="sub-header font-weight-bold">
+        {{ $t('Currently uploaded files') }}
+      </v-subheader>
+      <v-row>
+        <v-chip-group class="ml-5 mb-3">
+          <v-chip
+            v-for="(item, index) in completeApplication.uploadedDocuments"
```
Comment (sgaddameedi-insight) [inline UI/libs/core-public/src/lib/components/form-stepper/form-steps/FileUploadStep.vue:14]: Would it be easier if we add the chip on respective file inputs column?

**Review by sgaddameedi-insight — CHANGES_REQUESTED** [review]

File: UI/libs/core-public/src/lib/components/form-stepper/form-steps/FileUploadStep.vue (hunk 1 above)
Comment (jeremiahbooth-insight) [reply UI/libs/core-public/src/lib/components/form-stepper/form-steps/FileUploadStep.vue:14]: easier no. that would require a new function to loop and  probably add a state for each input to either change an icon or change the value of the field which would then cause the handle submit to change

Comment (sgaddameedi-insight) [reply UI/libs/core-public/src/lib/components/form-stepper/form-steps/FileUploadStep.vue:14]: Oh, how does it look right now? screenshot?

Comment (sgaddameedi-insight) [reply UI/libs/core-public/src/lib/components/form-stepper/form-steps/FileUploadStep.vue:14]: ok try this for file input


```
<template #selection="{ index }">
      <v-chip
        v-if="index < 2"
        color="blue1"
        dark
        label
        small
      >
           {{ 'Document file name goes here that you rename it to while upload' }}
      </v-chip>
</template>
```

**Pull request closed without merging**